web-app/src/index.js

```diff
@@ -35,10 +36,13 @@ const configureStore = () => {
 
 const isAuthenticated = () => {
   if (!authenticated()) {
-    return <Redirect to="/app" />;
+    if (window.location.pathname !== '/landingpage' && window.location.pathname !== '/develop' && window.location.pathname !== '/app' && window.location.pathname !== '/login' && window.location.pathname.substring(0, 25) !== '/login/callback/facebook/' && window.location.pathname !== '/reset-password' && window.location.pathname.substring(0, 16) !== '/reset-password/' && window.location.pathname !== '/sign-up' && window.location.pathname !== '/you-made-it' && window.location.pathname !== '/start') {
```

Comment:
Still some remarks

- If someone/we create a new page in the react-app, we will always need to add it into this array.
- Landingspage and onepager are now served by the Laravel app. This won't work for gh-pages. There is no guarantee that the server will exist in the future.
- Currently we are loading the full ReactJS app. Then we'll completely redirect to another page.